src/com/firebirdberlin/nightdream/ui/NightDreamUI.java

```diff
@@ -744,6 +768,59 @@ private void setupBackgroundImage() {
        }
     }
 
+    private double convertGPS(String[] separated){
+        double convert;
+        String[] separated2 = separated[2].split("/");
+        convert = Double.parseDouble(separated2[0]) / Double.parseDouble(separated2[1]) / 60;
+        String[] separated1 = separated[1].split("/");
+        convert = (Double.parseDouble(separated1[0]) + convert) / Double.parseDouble(separated1[1]) / 60;
+        String[] separated0 = separated[0].split("/");
+        convert = Double.parseDouble(separated0[0]) + convert;
+        return convert;
+    }
+
+    private String getExifInformation() throws IOException {
+        if (preloadBackgroundImageFile == null){
+            return "";
+        }
+
+        String exifDate ="";
+        String exifTime = "";
+        String exifCity = "";
+        String exifCountry = "";
+
+        ExifInterface exif = new ExifInterface(preloadBackgroundImageFile);
+
+        if (exif.getAttribute(ExifInterface.TAG_DATETIME) != null) {
+            String[] exifDateTime = exif.getAttribute(ExifInterface.TAG_DATETIME).split(" ");
+            String[] exifDateSplit = exifDateTime[0].split(":");
+            exifDate = exifDateSplit[2] + "." + exifDateSplit[1] + "." + exifDateSplit[0];
+            exifTime = exifDateTime[1];
+        }
```

Comment:
```suggestion
       String datetime = exif.getAttribute(ExifInterface.TAG_DATETIME);
        if (datetime != null) {
            String[] exifDateTime = datetime.split(" ");
            [...] 
        }
```
Ideally settings.dateformat and settings.getFullTimeFormat are used to format date and time.